subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3913,25 +3926,28 @@ static inline int length_req_rsp_recv(struct ll_conn *conn, memq_link_t *link,
 		/* use the minimal of our default_tx_octets and
 		 * peer max_rx_octets
 		 */
-		if (lr->max_rx_octets >= PDU_DC_PAYLOAD_SIZE_MIN) {
-			eff_tx_octets = MIN(lr->max_rx_octets,
+		if (sys_le16_to_cpu(lr->max_rx_octets) >=
```

Comment:
Use auto-variables to hold in machine endian and use in the if-clauses that check-assign to variables.

Reply:
Good point. Done.